apturicovid/viewController/StatsVC.swift

```diff
@@ -124,6 +130,17 @@ extension StatsVC : UICollectionViewDataSource {
         stats?.totalItemCount ?? 0
     }
     
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
+        if let headerView = self.collectionView(collectionView, viewForSupplementaryElementOfKind: UICollectionView.elementKindSectionHeader, at: IndexPath(row: 0, section: section)) as? StatsHeaderView {
+            
+            return headerView.systemLayoutSizeFitting(CGSize(width: collectionView.frame.width, height: UIView.layoutFittingCompressedSize.height),
+                                                      withHorizontalFittingPriority: .required,
+                                                      verticalFittingPriority: .fittingSizeLevel)
```

Comment:
@ULazdins UICollectionView's auto-layout can be achieved through code only. I've updated the method.